### PR TITLE
lit functionality

### PR DESCRIPTION
**Pull request by mikestephens**
https://github.com/hash-gaming/randbot/issues/8

Adds the ability to post to #lit from any channel that the bot is part of

### CODE REVIEW

File: scripts/lit.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Description:
+//   a collection of general gamebot functions
+//
+// Dependencies:
+//
+// Configuration:
+//
+// Commands:
+// lit
```
Comment (YashdalfTheGray) [inline scripts/lit.js]: So hubot uses this to generate help. Can we rephrase this to be something like
```
// Commands:
//   lit <message_link> - Immortalizes a message in #lit
```

File: scripts/lit.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// Description:
+//   a collection of general gamebot functions
```
Comment (YashdalfTheGray) [inline scripts/lit.js]: This is a total nit but change this too?

File: package.json [hunk 3]
```diff
@@ -1,29 +1,32 @@
...
+        "hubot-slack": "4.4.0"
+    },
+    "engines": {
+        "node": ">=8.0.0"
+    },
+    "devDependencies": {
+        "eslint": "4.8.0",
+        "eslint-config-airbnb-base": "12.0.2",
+        "eslint-plugin-import": "2.7.0"
+    }
```
Comment (mikestephens) [inline package.json]: looks like my linter did the 4 space

Comment (YashdalfTheGray) [reply package.json]: Yeah, and anytime we include a new npm package or do anything with `npm`, it's going to rewrite it in two spaces so can we go back to two?

File: scripts/lit.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+// Description:
+//   a helpful tool for posting #lit comments
+//
+// Dependencies:
+//
+// Configuration:
+//
+// Commands:
+// lit <message_link> - Immortalizes a message in #lit... fam
```
Comment (YashdalfTheGray) [inline scripts/lit.js]: Heh 💯

File: package.json [hunk 5]
```diff
@@ -1,29 +1,32 @@
...
-  }
+    "name": "randbot",
+    "version": "1.0.0",
+    "private": true,
+    "author": "hash-gaming",
+    "description": "Mostly productive but sometimes trolly",
+    "scripts": {
+        "linter": "eslint .",
+        "local-server-start": "docker-compose up --build --force-recreate",
+        "local-server-stop": "docker rm $(docker ps -a -q)",
```
Comment (YashdalfTheGray) [inline package.json]: I'd rather this be `"local-server-stop": "docker-compose down"` because that won't remove all docker containers on your computer which is only needed as a cleanup step once you know that docker containers on your computer aren't needed anymore. `docker-compose down` just cleans up the resources that `docker-compose up` created.

File: package.json (hunk 3 above)
Comment (mikestephens) [reply package.json]: 🎉
